src/components/SuccessNotification.tsx

```diff
@@ -0,0 +1,39 @@
+import { Transition } from '@headlessui/react';
+import { CheckCircleIcon, XMarkIcon } from '@heroicons/react/20/solid';
+import { useSuccess } from '~/data/hooks/success';
+
+export default function SuccessNotification() {
+  const { success, setSuccess } = useSuccess();
+
+  return (
+    <Transition show={success !== null}>
```

Comment:
could we make the success notification clear itself after a short period? say 2-3 seconds? i think success notifications should be short-lived, while error notifications should persist until manually closed/acknowledged. wyt?

Reply:
This is a good thought, I wasn't sure if this was a desired functionality.  Set timeout to be 3 seconds for now.